### PR TITLE
CPWZ sensor logger changes

### PR DESCRIPTION
**Pull request by mnaguit**
- from Q330/3 (2551) to Centaur (8748)
- new sensor for LC 20: Guralp Fortis (TF1130)
- open LC 20 for HN-ENZ
- update metadata for LC 10 (EH-ENZ)
- Centaur firmware v4.5.20

### CODE REVIEW

File: install/connections.csv [hunk 1]
```diff
@@ -121,6 +121,7 @@ CPE,10,Cape Palliser,Short Period,,1990-10-05T11:52:00Z,1990-10-07T19:26:00Z
 CPIT,40,Castlepoint,,0,2009-09-16T00:00:00Z,9999-01-01T00:00:00Z
 CPIT,41,Castlepoint,,1,2009-09-16T00:00:00Z,9999-01-01T00:00:00Z
 CPWZ,10,Castlepoint Station,,,2009-07-07T01:00:00Z,9999-01-01T00:00:00Z
+CPWZ,20,Castlepoint Station,,,2023-02-10T02:04:00Z,9999-01-01T00:00:00Z
```
Comment (salichon) [inline install/connections.csv:124]: as in https://github.com/GeoNet/delta/blob/2bb3002d7b19a5766bf92eeec07af3f37b73dcdc/install/README.md#connections

It s better (and future)  practice to set the connection `number` (pins) 
for the datalogger with multi-sensors i reckon.  Here 0 and 3  
thanks

**Review by salichon — CHANGES_REQUESTED** [review]
pin/number to amend -to me

File: install/connections.csv (hunk 1 above)
Comment (mnaguit) [reply install/connections.csv]: pin number inserted

**Review by CBurton90 — APPROVED** [review]

**Comment by mnaguit** [comment]
@salichon changes made for `connections.csv`, thanks

**Review by salichon — APPROVED** [review]
Purfect thanks @mnaguit :)

**Pull request merged**